core/lib/src/local/blocking/client.rs

```diff
@@ -88,6 +88,13 @@ impl Client {
     pub_client_impl!("use rocket::local::blocking::Client;");
 }
 
+impl Drop for Client {
+    fn drop(&mut self) {
+        let client = self.inner.take();
```

Comment:
Instead of making this an `Option`, so we can `take()`, can we simply `swap` in a new `asynchronous::Client` and drop the old one in `self.block_on()`? This would avoid all of the `as_ref().unwrap()` noise above.

Reply:
That was my first idea, but swapping in a new `asynchronous::Client` first requires constructing an entire `Rocket` instance.